src/coreComponents/mesh/generators/NewGhosting.cpp

```diff
@@ -898,10 +1086,13 @@ std::tuple< MeshGraph, GhostRecv, GhostSend > performGhosting( MeshGraph const &
   //   knowing the index of the nodes is not enough.
   //
   // In order to do that, we build the `missingIndicator` matrix, which is rectangular:
-  // - The number of columns is the number of graph nodes in the mesh graph.
+  // - The number of columns is the number of graph nodes in the mesh graph, ie the total number of geometrical entities in the mesh.
   // - The number of rows is the total number of graph nodes that are missing on ranks.
+  //   For me this was confusing, as its a little different from what we've done before
```

Comment:
We could have handled it with direct MPI communications instead of using linear algebra. Up to you.